hoot-core/src/main/cpp/hoot/core/algorithms/ExpectationIntersection.cpp

```diff
@@ -122,7 +122,7 @@ vector<double> ExpectationIntersection::snapMatches(const Mat& matches, const Ma
   int iteration = 0;
   bool done = false;
 
-  while (!done && iteration++ < _maxIterations)
+  while (!done && iteration < _maxIterations)
```

Comment:
This PR is mostly trying out new `git` commands so I fixed a few `blocker` issues from the Qube.  Here it is removing anything that modifies a variable on the right hand side of the `&&` operator because when `!done` fails the right hand side isn't executed.  Here in these instances that side effect doesn't matter.  So I fixed them all.